build_system/src/clean.rs

```diff
@@ -0,0 +1,56 @@
+use crate::utils::remove_file;
+
+use std::fs::remove_dir_all;
+
+fn check_args() -> Result<Option<()>, String> {
+    for arg in std::env::args().skip(2) {
```

Comment:
I always like a comment to mention that this skips the binary name and the command `clean`.